server/server.ts

```diff
@@ -2,16 +2,16 @@ import express from "express";
 import cookieParser from "cookie-parser";
 
 import db from "./models/index";
-import userRouter from "./routes/v1/user";
+import v1Routes from "./routes/v1";
 
 const app = express();
 const port = 8080;
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-app.use(cookieParser(process.env.SECRET));
+app.use(cookieParser(process.env.COOKIE_SECRET));
```

Comment:
```suggestion
app.use(cookieParser());
```
Since the JWT is signed, I don't think this is necessary. Also update README.md